circle.yml

```diff
@@ -0,0 +1,8 @@
+test:
+  override:
+    - "true"
```

Comment:
Why do you need this? Normally, CircleCI's inference should run `npm test` which, as it is undefined, will simply `echo 'No test specified'` and succeed.